torchci/pages/nightlies.tsx

```diff
@@ -0,0 +1,453 @@
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import dayjs from "dayjs";
+import ReactECharts from "echarts-for-react";
+import { EChartsOption } from "echarts";
+import {
+  GridRenderCellParams,
+  GridValueFormatterParams,
+} from "@mui/x-data-grid";
+import useSWR from "swr";
+import {
+  Grid,
+  Paper,
+  TextField,
+  Typography,
+  Stack,
+  Skeleton,
+  Select,
+  MenuItem,
+  FormControl,
+  InputLabel,
+  SelectChangeEvent,
+} from "@mui/material";
+import { LocalizationProvider, DateTimePicker } from "@mui/x-date-pickers";
+
+import { useEffect, useState } from "react";
+
+import { RocksetParam } from "lib/rockset";
+import { fetcher } from "lib/GeneralUtils";
+
+import TablePanel from "components/metrics/panels/TablePanel";
+import { durationDisplay } from "components/TimeUtils";
+
+function NightlyJobsRedPanel({ params, repo }: { params: RocksetParam[], repo: string }) {
+
+  let repo_p = params.find(({ name }) => name == "repo");
+  if(repo_p && repo) repo_p.value = repo; 
+
+  const url = `/api/query/nightlies/nightly_jobs_red?parameters=${encodeURIComponent(
+    JSON.stringify(params)
+  )}`;
+
+  const { data } = useSWR(url, fetcher, {
+    refreshInterval: 5 * 60 * 1000, // refresh every 5 minutes
+  });
+
+  if (data === undefined) {
+    return <Skeleton variant={"rectangular"} height={"100%"} />;
+  }
+
+  const options: EChartsOption = {
+    title: { text: "% "+repo+" nightly jobs by red" },
```

Comment:
Nit: `by red` sounds weird to me, may be just call it `% vision night jobs failures`.  Also the current page display all 4 repos, another way to display the information is to have a drop down list with `pytorch`, `vision`, `audio`, and `test`, and may be `all`.  The latter might be more useful when the number of repos increases, but it's not a big deal atm.

Reply:
For now, we want to avoid dropdown. But display all the repo failures at once since all of these are part of the release and increase of failures on any of the repo should be noted